README.rst

```diff
@@ -1023,6 +1023,17 @@ jitter, datagram loss etc of the device using `iperf3 utility <https://iperf.fr/
 This check is **disabled by default**. You can enable auto creation of this check by setting the
 `OPENWISP_MONITORING_AUTO_IPERF <#OPENWISP_MONITORING_AUTO_IPERF>`_ to ``True``.
 
+You can also create the iperf check from the **device page** as shown below:
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-check.png
+  :align: center
+
+**Note:** To access the above page, the user must have some special permissions,
+which are included by default in the groups "Administrator" and "Operator" and are shown in the screenshot below.
+
+.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/inline-permissions.png
+  :align: center
+
```

Comment:
Shall we mention also how to create the alert setting and indicate how to get alerted if the bandwidth is less than, for example, 30 Mmbit/s?

Reply:
Yea, for this we have to change `MetricInline` visible fields to include `field_name` as well (we may add it in advance mode),  For example:
```py
    fieldsets = [
        (None, {'fields': ('is_healthy', 'configuration')}),
        (
            _('Advanced options'),
            {'classes': ('collapse',), 'fields': ('field_name',)},
        ),
    ]
```

![Screenshot from 2022-09-03 16-42-56](https://user-images.githubusercontent.com/56113566/188267848-791b3b9d-3feb-4acc-9e5c-23b45f175f5a.png)

@nemesisdesign I think we should add this in https://github.com/openwisp/openwisp-monitoring/pull/427 as you've already suggested in https://github.com/openwisp/openwisp-monitoring/pull/427#discussion_r951633955, what do you think?

Reply:
Yes, this would work, as long as the field name defaults to the one configured and as long as we do validation of this value before saving to ensure it exists.

Reply:
> Yes, this would work, as long as the field name defaults to the one configured and as long as we do validation of this value before saving to ensure it exists.

@nemesisdesign I think we should add this in https://github.com/openwisp/openwisp-monitoring/pull/427 as you've already suggested in https://github.com/openwisp/openwisp-monitoring/pull/427#discussion_r951633955, what do you think?

Reply:
Yes, right, ok!